src/NewRemotingUnitTest/RemoteOperationsTest.cs

```diff
@@ -2,47 +2,63 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
-using System.Linq;
-using System.Management;
 using System.Reflection;
 using System.Runtime.Serialization;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Castle.DynamicProxy;
 using NewRemoting;
 using NUnit.Framework;
 using SampleServerClasses;
 
 namespace NewRemotingUnitTest
 {
-	[TestFixture]
+	[TestFixture(true)]
+	[TestFixture(false)]
 	public class RemoteOperationsTest
 	{
 		private Process _serverProcess;
 		private Client _client;
 		private string _dataReceived;
 		private string _dataReceived2;
+		private AuthenticationHelper _helper;
 
-		[OneTimeSetUp]
-		public void OneTimeSetUp()
+		public RemoteOperationsTest(bool withAuthentication)
+		{
+			if (withAuthentication)
+			{
+				_helper = new AuthenticationHelper();
+			}
+		}
+
+		public AuthenticationInformation CreateClientServer()
 		{
 			FileInfo fi = new FileInfo(Assembly.GetExecutingAssembly().Location);
 			var psi = new ProcessStartInfo(Path.Combine(fi.DirectoryName, "RemotingServer.exe"));
 
 			Console.WriteLine($"Attempting to start {psi.FileName}...");
 			Console.WriteLine($"Current directory is {Environment.CurrentDirectory}");
+			AuthenticationInformation authenticationInfo = _helper == null ? null : new AuthenticationInformation(_helper.CertificateFileName, _helper.CertificatePassword);
+
 			_serverProcess = Process.Start(psi);
 			Assert.IsNotNull(_serverProcess);
 
 			// Port is currently hardcoded
-			_client = new Client("localhost", Client.DefaultNetworkPort, null);
+			_client = new Client("localhost", Client.DefaultNetworkPort, authenticationInfo, null, null);
 			_client.Start();
+			return authenticationInfo;
+		}
+
+		[SetUp]
+		public void SetUp()
```

Comment:
it might not be required but I think it makes test independent and some tests like (UseSystemManagementLocally, TwoClientsCanConnect) do not require the remoting server to run.